rstar/src/test_utilities.rs

```diff
@@ -4,6 +4,8 @@ use rand::distributions::Uniform;
 use rand::{Rng, SeedableRng};
 use rand_hc::Hc128Rng;
 
+use crate::lib::*;
```

Comment:
I believe this import shouldn't be required as `test_utilities` is only used with the `test` feature.

Reply:
Good catch!